src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
```

Comment:
You can immediately return the object instead of constantly duplicating `calculator.`
```suggestion
  return { ..... };
```